docs/pages/access-controls/access-monitoring.mdx

```diff
@@ -0,0 +1,208 @@
+
+
+## Prerequisites
+- Teleport Cloud cluster v14.2.2 or later.
+
+## Access Monitoring RBAC Permissions
+
+To access the Access Monitoring interface, a user must have a role that allows
+`list`, `read` and `use` verbs on the `audit_query` and `audit_query` resources.
+By default the built-in `auditor` role has these permissions or you can create a custom role with these permissions e.g:
+```yaml
+kind: role
+metadata:
+  name: my-role
+spec:
+  allow:
+    rules:
+    - resources:
+      - security_report
+      verbs:
+      - list
+      - read
+      - use
+    - resources:
+      - audit_query
+      verbs:
+      - list
+      - read
+      - use
+```
+
+## Query Editor
+The Query Editor in Teleport Access Monitoring provides users with a powerful interface to interactively query audit logs and generate insightful reports.
+It offers a user-friendly environment for constructing SQL queries tailored to specific use cases,
+allowing users to gain deep insights into access patterns and security events within Teleport cluster.
+
+Within the Query Editor, users have access to a diverse range of SQL views representing various audit events captured by Teleport.
+These views provide a comprehensive overview of activities within the Teleport cluster, enabling users to construct precise queries tailored to their monitoring needs.
+Below is a list of the available SQL views:
+
+```
+access_list_create
+access_list_delete
+access_list_member_create
+access_list_member_delete
+access_list_member_update
+access_list_review
+access_list_update
+access_request_create
+access_request_review
+auth
+bot_join
+cert_create
+db_session_query
+db_session_query_failed
+db_session_start
+device_authenticate
+device_enroll
+exec
+instance_join
+join_token_create
+kube_request
+lock_created
+lock_deleted
+recovery_code_used
+reset_password_token_create
+saml_idp_auth
+session_command
+session_join
+session_rejected
+session_start
+user_create
+user_login
+user_password_change
+windows_desktop_session_end
+windows_desktop_session_start
+```
+
+To access the Query Editor, navigate to the `Access Monitoring` section in the Teleport UI and click on the `Query Editor` tab.
+
+
+### Here are a few query examples that demonstrate the capabilities of Access Monitoring
+
+- Query for unique users who executed a ssh command involving `/etc/passwd` file in some way:
+```sql
+SELECT
+  DISTINCT user
+FROM
+  exec
+WHERE
+  command LIKE '%/etc/passwd%';
+```
+
+![exec passwd](../../img/access-monitoring/exec_passwd.png)
+
+- Select the count of unique IP addresses associated with each user cert over different event dates:
+```sql
+SELECT
+  event_date,
+  identity_user AS user,
+  COUNT(DISTINCT cert_create.identity_client_ip) AS unique_ip_count
+FROM
+  cert_create
+GROUP BY
+  event_date,
+  identity_user
+ORDER BY
+  event_date;
+```
+
+![cert ip](../../img/access-monitoring/cert_create_ip.png)
```

Comment:
I think we probably only need to show one screenshot. We can list multiple example queries in the docs, but all of these screenshots are basically the same (a query with a table showing the results)